conflict.py

```
@@ -1,144 +1,267 @@
-    if confirm == 'y':
-        # Note: Actual conflict storage will occur once courses exist
-        print("Conflict recorded.")
-    else:
-        print("Conflict addition canceled.")
\ No newline at end of file
+
+# Filename: conflict.py
+# Description: Functions to add, delete, modify conflicts
+# Authors: Lauryn Gilbert, Luke Leopold
+
+from scheduler import *
```

Ruff (0.15.0)

[error] `from scheduler import *` used; unable to detect undefined names

(F403)